examples/sensors-viewer/src/components/FlashHomeModal.jsx

```diff
@@ -0,0 +1,60 @@
+import React, {Component} from 'react';
+import {Button, Col, Form, Modal} from "react-bootstrap";
+import {setAuthToken, setRealmName} from "../controller/storageService";
+
+class FlashHomeModal extends Component {
+  state = {
+    realm_name: '',
+    token: '',
+  };
+
+  handleRealm = (e) => {
+    e.preventDefault();
+    this.setState({[e.target.name]: e.target.value});
+  };
+
+  handleSubmit = () => {
+    const {token, realm_name} = this.state;
+    setAuthToken(token);
+    setRealmName(realm_name);
+    this.props.handleModal()
```

Comment:
handleModal() doesn't close the popup, checkout Modal at https://react-bootstrap.github.io/components/modal/

Reply:
This is the prop i passed to the child component, its not of the react-bootstrap.